strutil/strutil.go

```diff
@@ -40,3 +42,11 @@ func JSON(v interface{}) string {
 	bytes, _ := json.Marshal(v)
 	return string(bytes)
 }
+
+// ForwardSlashify replaces all '\' with '/' on Windows OS
+func ForwardSlashify(str string) string{
```

Comment:
If I'm not missing anything we might be able to use [`filepath.ToSlash()` ](https://golang.org/pkg/path/filepath/#ToSlash)

> ToSlash returns the result of replacing each separator character in path with a slash ('/') character. Multiple separators are replaced by multiple slashes.

Reply:
thanks! yes it seems filepath.ToSlash() does the job.